pkgs/applications/science/logic/coq/default.nix

```diff
@@ -118,7 +119,12 @@ self = stdenv.mkDerivation {
      then [ ocamlPackages.lablgtk3-sourceview3 glib gnome3.defaultIconTheme wrapGAppsHook ]
      else [ ocamlPackages.lablgtk ]);
 
-  propagatedBuildInputs = stdenv.lib.optional (versionAtLeast "8.12") ocamlPackages.num;
+  propagatedBuildInputs =
+    if versionAtLeast "8.13"
```

Comment:
Please us lib.optional instead of ``if a then [ b ] or []``.

Reply:
Rationale?

Reply:
Fixed.